src/vivarium_census_prl_synth_pop/constants/metadata.py

```diff
@@ -235,6 +244,10 @@
     "housing_type",
 ]
 
+NATIVITY_MAP = {1: True, 2: False}
```

Comment:
What's the meaning of these keys and values for `NATIVITY_MAP` and `MIGRATION_MAP`?

Reply:
@mattkappel the input data we use from ACS is survey data so all responses have a code and isn't human readable in a sense.  This map is for a question asking respondents if they were born in the United States.

Reply:
A comment would be maybe appropriate, as @zmbc suggests to link to the data dictionary?